### ISSUE TITLE
Use the Prometheus exposition format for exposing metrics

### ISSUE THREAD

**Issue by leecalcote** [layer5io/learn-layer5#18]
Does it make sense to instrument using https://github.com/prometheus/client_golang?  // @kanishkarj

_Originally posted by @leecalcote in https://github.com/layer5io/learn-layer5/pull/15/files_

**Comment by kanishkarj** [comment]
Yes, it is. It's always better to follow a standardized format I guess, xD

**Comment by kanishkarj** [comment]
I'm curious if we can expose JSON response via the prometheus instrumentation libraries. I think it supports only counts and histogram.
